op-batcher/batcher/channel_builder.go

```diff
@@ -12,7 +12,8 @@ import (
 )
 
 var (
-	ErrInvalidMaxFrameSize   = errors.New("max frame size cannot be zero")
+	ErrZeroMaxFrameSize      = errors.New("max frame size cannot be zero")
+	ErrSmallMaxFrameSize     = errors.New("max frame size cannot be less than 23")
```

Comment:
Error variables `Err...` only make sense if there’s a real chance that the caller receiving this error might react differently to it than to other errors. Otherwise they just clutter the package’s API. In this case here they would be returned by the config check functions and then startup of the application would fail. I don't see how any caller, at runtime, would react differently to these. So I'm in favor or removing these.